src/openforms/forms/constants.py

```diff
@@ -55,3 +55,7 @@ class FormVariableDataTypes(DjangoChoices):
 
 class FormVariableStaticInitialValues(DjangoChoices):
     now = ChoiceItem("now", _("Now"))
+    auth_identifier = ChoiceItem("auth_identifier", "Authentication identifier")
+    auth_bsn = ChoiceItem("auth_bsn", "Authentication BSN")
+    auth_kvk = ChoiceItem("auth_kvk", "Authentication KvK")
+    auth_pseudo = ChoiceItem("auth_pseudo", "Authentication pseudo")
```

Comment:
do we need this? is this going to be annoying to extend if other apps define their own static variables? It feels like this is superseded by the variables registry

Reply:
Yes, it doesn't look to be used anywhere anymore